man/adoc/bpftrace.adoc

```diff
@@ -1045,6 +1045,12 @@ For string arguments use the `str()` call to retrieve the value.
 | ktime_get_ns / ktime_get_boot_ns
 | nanoseconds since kernel boot. On kernels that support `ktime_get_boot_ns` this includes the time spent suspended, on older kernels it does not.
 
+| jiffies
+| uint64
+| 5.6
```

Comment:
Technically speaking, it's only possible to use this from tracing programs from 5.9 (https://github.com/torvalds/linux/commit/72e2b2b66f9c).

Reply:
Aha yes, I just looked at the https://github.com/torvalds/linux/commit/5576b991e9c1a11d2cc21c4b94fc75ec27603896 and thought it was introduced in 5.6, thanks for pointing out ;)